HLTrigger/HLTanalyzers/test/CreateCFGs.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/python
 #
 
+from builtins import range
```

Comment:
Also here: Why is this single import needed - without any other change? Is it overriding a default?